content/en/docs/setup/install/istioctl/index.md

```diff
@@ -388,17 +394,20 @@ which allows the following settings to be changed. Use this list to identify the
 1. [Priority class name](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass)
 1. [Node selector](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector)
 1. [Affinity and anti-affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)
+1. [Service](https://kubernetes.io/docs/concepts/services-networking/service/)
+1. [Toleration](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)
+1. [Strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
+1. [Env](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/)
 
 All of these Kubernetes settings use the Kubernetes API definitions, so [Kubernetes documentation](https://kubernetes.io/docs/concepts/) can be used for reference.
 
-The following example overlay file adjusts the `TrafficManagement` feature's resources and horizontal pod autoscaling
+The following example overlay file adjusts the resources and horizontal pod autoscaling
 settings for Pilot:
 
 {{< text yaml >}}
-apiVersion: install.istio.io/v1alpha2
-kind: IstioControlPlane
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
 spec:
-  trafficManagement:
```

Comment:
When you delete this line, you need to unindent (by 2 spaces) all the following lines in the structure.

Reply:
done